main.go

```diff
@@ -82,6 +83,7 @@ func parseFlags() (*config, error) {
 	flag.StringVar(&cfg.oidc.audience, "oidc.audience", "", "The audience for whom the access token is intended, see https://openid.net/specs/openid-connect-core-1_0.html#IDToken.")
 	flag.StringVar(&cfg.oidc.username, "oidc.username", "", "The username to use for OIDC authentication. If both username and password are set then grant_type is set to password.")
 	flag.StringVar(&cfg.oidc.password, "oidc.password", "", "The password to use for OIDC authentication. If both username and password are set then grant_type is set to password.")
+	flag.StringSliceVar(&cfg.scope, "scope", []string{""}, "The scope to be included in the payload data of the token.  Scopes can either be comma-separated or space-separated.")
```

Comment:
```suggestion
	flag.StringSliceVar(&cfg.scope, "scope", []string{""}, "The scope to be included in the payload data of the token. Scopes can either be comma-separated or space-separated.")
```

Reply:
@lindseyburnett I don't think that the values to the flag can be space separated. The shell will interpret the second or third scope as arguments to the CLI. Perhaps they can be space-separated if the scopes are wrapped in quotes to indicate that it's one single value. pflag definitely allows for comma-separation _and_ specifying the flag multiple times, e.g. `--scope X --scope Y --scope Z`

Reply:
Yup, as per `StringSliceVar` goDoc comment all of these are valid. :)